data/tools/GUI.pyw

```diff
@@ -268,7 +278,8 @@ Self destroys when the tool thread is over"""
         frame=Frame(self)
         frame.pack(fill=BOTH,expand=YES)
         wait_label=Label(frame,
-                         text="{0} is running\nPlease wait...".format(tool),
+                         # TRANSLATORS: {0} is the name of command being executed.
```

Comment:
This translation comment isn't appearing the generated .pot file.

Reply:
I haven't pushed the new .pot file after adding this comment yet. Did you test it?

Reply:
12d57c090b87d79bca3e0d5618fdf63ff3d6f635

Reply:
I didn't test it, I assumed the PR would show what it was currently generating.